### PR TITLE
Create a db-restore command

### PR DESCRIPTION
**Pull request by SamuelDavis**
https://www.pivotaltracker.com/story/show/121222671

### CODE REVIEW

**Comment by maedayx** [comment]
🍰

File: src/commands/DbCommand.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        }
+        return [
+            'user' => Arr::pluck($env, ['DB_USER', 'DB_USERNAME', 'MYSQL_USER', 'MYSQL_USERNAME']),
+            'pass' => Arr::pluck($env, ['DB_PASS', 'DB_PASSWORD', 'MYSQL_PASS', 'MYSQL_PASSWORD']),
+            'host' => Arr::pluck($env, ['DB_HOST', 'MYSQL_HOST']),
+            'name' => Arr::pluck($env, ['DB_NAME', 'DB_DATABASE', 'MYSQL_NAME', 'MYSQL_DATABASE']),
+        ];
+    }
+
+    protected function exec($cmd)
```
Comment (sudo-reboot) [inline src/commands/DbCommand.php]: this doesn't have anything to do with DB stuff, could be in base command class or in a helper

File: src/commands/DbBackup.php [hunk 2]
```diff
@@ -1,72 +1,39 @@
...
-                static::ARG_PROJECT,
-                InputArgument::REQUIRED,
-                'Project name (reference ECS service name)'
-            )
-            ->addArgument(
-                static::ARG_ENV,
-                InputArgument::REQUIRED,
-                'Project environment (reference ECS service name)'
-            );
+        parent::configure()->setDescription('Download a mysqldump of a project.');
```
Comment (sudo-reboot) [inline src/commands/DbBackup.php]: should probably return here for consistency

File: src/commands/DbRestore.php [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+class DbRestore extends DbCommand
+{
+    const ARG_FILE = 'file';
+
+    private $ecsGateway;
+
+    protected function configure()
+    {
+        parent::configure()
```
Comment (sudo-reboot) [inline src/commands/DbRestore.php]: also here

File: src/commands/DbRestore.php [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+        $db = $this->buildDbConfig($taskDefinition);
+        $connectCmd = implode(' ', [
+            'mysql',
+            "-u {$db['user']}",
+            "-p{$db['pass']}",
+            "-h {$db['host']}",
+
+        ]);
+        $dropCmd = "{$connectCmd} -e \"DROP DATABASE {$db['name']}; CREATE DATABASE {$db['name']};\"";
+        $restoreCmd = "{$connectCmd} {$db['name']} < {$input->getArgument(static::ARG_FILE)}";
```
Comment (sudo-reboot) [inline src/commands/DbRestore.php]: you should verify this file exists before dropping the table

**Pull request merged**